### PR TITLE
Some general polish in TS

### PR DESCRIPTION
**Pull request by abcdefg30**
Split from #7967 as wished by @Mailaender.

### CODE REVIEW

**Comment by Mailaender** [comment]
:+1:

**Comment by abcdefg30** [comment]
Also added `BuildLimit: 1` to the `Commando Cyborg` now.

File: mods/ts/rules/gdi-vehicles.yaml [hunk 1]
```diff
@@ -149,6 +148,7 @@ HMEC:
 		Queue: Vehicle
 		BuildPaletteOrder: 80
 		Prerequisites: ~gaweap, gatech
+		BuildLimit: 1
```
Comment (pchote) [inline mods/ts/rules/gdi-vehicles.yaml]: The description should be updated to say "Maximum 1 can be trained".

Comment (pchote) [inline mods/ts/rules/gdi-vehicles.yaml]: Nevermind, that's covered in #7967.

**Comment by pchote** [comment]
I noticed a few more polish nits while testing this, but they don't need to hold up this existing batch :+1:

**Pull request merged**